lib/trie/trie_test.go

```diff
@@ -3548,3 +3548,40 @@ func Benchmark_concatSlices(b *testing.B) {
 		}
 	})
 }
+
+// ParityTech/trie proofs are incompatible with gossamer/lib/trie #2329
+func TestTrieProofParityTechCompatible(t *testing.T) {
+	//TODO: enable it once #2329 is fixed
+	t.Skip("NOT YET FIXED")
```

Comment:
should this be fixed now?

Reply:
Hello @timwu20, thanks for reaching out. 
This PR does not intend to provide a fix for the issue. IMHO the fix should come as part of a resolution for the following issue: https://github.com/ChainSafe/gossamer/issues/2329 ; I understand that @qdm12 and @EclesioMeloJunior are on top of the problem already.

Reply:
I'm working on upgrading the trie.
Let's keep this PR opened for now, and we can merge it before the trie upgrade so I can use those values in my branch.